application/templates/pages/about/performance.md

```diff
@@ -0,0 +1,35 @@
+Performance indicators showing how the planning data platform is making it easier to find, use and trust planning and housing data in England.
+
+Last updated 19 July 2024.
+
+## Availability
+
+### 89 datasets on the platform
+
+**00 national datasets** available from **00 organisations**. 0 added last month (+/-0%).
+
+**00 datasets** provided by **00 local planning authorities**. 0 provided last month (+/-0%).
+
+0 new authorities providing data. +/-0% change from last week.
+
+## Quality
+
+### 00 out of 00 quality score
+
+00 datasets with no issues (0%). 00 fixed last week (+/-0%).
+
+00 datasets conform to the specifications (0%). 00 improved last week (+/-0%).
+
+00 datasets up to date (0%). 00 updated last week (+/-0%).
```

Comment:
I think “up to date” consists of two separate things: 

* whether we’ve reflected any changes to an endpoint on the site
* and also whether an endpoint has been updated at all within an expected timeframe. 

Maybe it’s worth breaking out? The first is in our control but the second isn’t. So could be like: 
* “00 datasets in sync with site”
* “00 datasets updated within last [expected timeframe]”

Reply:
Hmm, I see what you mean, although I think those are perhaps internal metrics? Knowing how many endpoints have been updated and have new data to index is useful to us, but to data consumers it just communicates that we have a pipeline and some things are in a queue. I'd assume what they care about mostly is whether things are up-to-date or not, i.e. the queue has been processed.

Endpoints being up-to-date within the expected timeframe is what I mean by `up to date` here, as I assume that matters more to data consumers.